File: src/Domain/Entities/Gig.cs
using System;

namespace Domain.Entities
{
    public class Gig
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public TimeSpan Time { get; set; }
        public int BandId { get; set; }
        public int CafeId { get; set; }

        public virtual Band Band { get; set; }
        public virtual Cafe Cafe { get; set; }
    }
}

File: src/web-ui/src/app/models/gig.ts
export interface IGigsListVm {
  gigs: IGig[];
}

export interface IGig {
  id: number;
  date: Date;
  time: string;
  bandId: number;
  cafeId: number;
  band: IBand;
  cafe: ICafe;
}

interface IBand {
  id: number;
  name: string;
  genre: string;
}

interface ICafe {
  id: number;
  name: string;
  city: string;
  address: string;
}
